samples/charts/category-chart/marker-options/Pages/CategoryChartMarkerOptions.razor

```diff
@@ -0,0 +1,191 @@
+﻿@page "/CategoryChartMarkerOptions"
+
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <span className="igOptions-label">Chart Type: </span>
+        <select @onchange="OnChartTypeChanged">
+            <option>Auto</option>
+            <option>Area</option>
+            <option>Column</option>
+            <option>Point</option>
+            <option>Line</option>
+            <option>Spline</option>
+            <option>SplineArea</option>
+            <option>StepArea</option>
+            <option>StepLine</option>
+            <option>Waterfall</option>
```

Comment:
perpend all values of options with "CategoryChartType." this way, it will simply OnChartTypeChanged()